lm10/spiders/filings.py

```diff
@@ -480,6 +482,57 @@ def _get_i_value(cls, tree, label_text):
             result = tree.xpath(nonnormal_i_value_xpath)
 
         return result.get(default="")
+    
+    @classmethod
+    def _get_12b(cls, activity, table):
+        result = activity
+
+        result["federal_work"] = 'Field not present'
```

Comment:
let's have these be `None`s 

and let's have an additional boolean value of whether 12b exists or now.

Reply:
done! i moved these fields up into the `_activity_details` method with the other fields. and im using a conditional that checks the new `12b_exists` boolean to run the method that assigns values